### PR TITLE
Implementation of EditCurrentBody in LogicManager.

### LINKED ISSUE #401: Implement EditCurrentBlockBody in LogicManager
For UI to save/edit the block body.

### PR DESCRIPTION
**Pull request by ljiazh3ng**
resolves #401 
Due to the change in implementation. 
User will now be able to edit the block body using the MarkdownModel, no longer using the edit command.
This PR:
- [x] provides a method in the `LogicManager` to edit the currentBlockBody.
- [x] LogicMangerTest

### CODE REVIEW

File: src/main/java/com/notably/logic/LogicManager.java [hunk 1]
```diff
@@ -52,6 +55,15 @@ public void execute(String commandText) throws CommandException, ParseException
         }
     }
 
+    @Override
+    public void editCurrentBlockBody(String bodyContent) throws CannotModifyRootException {
```
Comment (kevinputera) [inline src/main/java/com/notably/logic/LogicManager.java]: `CannotModifyRootException` is a `RuntimeException`, so you won't need to declare `throws` here!

If you want, you can create a custom checked exception in the `com.notably.logic.exceptions` package. We can call it something like `EditBlockBodyException`, for example!

We can then catch the `CannotModifyRootException` and rethrow an `EditBlockBodyException`!

**Review by kevinputera — APPROVED** [review]
LGTM! Just one minor comment

**Comment by kevinputera** [comment]
Oh yes, @ljiazh3ng , let's add a small test case for this, shall we?

File: src/main/java/com/notably/logic/LogicManager.java (hunk 1 above)
Comment (ljiazh3ng) [reply src/main/java/com/notably/logic/LogicManager.java]: Ah i was thinking whether i should create a exception class for this sole purpose. Ok i'll do that

**Comment by ljiazh3ng** [comment]
> Oh yes, @ljiazh3ng , let's add a small test case for this, shall we?

Will add a LogicManagerTest!

**Pull request merged**